<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 194, 194);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>2</y>
      <width>280</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logoLM">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo/images/logo.jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo/images/search.png);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="H_Homepage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Homepage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H_KatalogProduk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Katalog Produk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H_Konten">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Blog/Konten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H_Event">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H_ForumDiskusi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Forum Diskusi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>239</y>
      <width>1161</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255,255);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Edit Produk</string>
    </property>
   </widget>
   <widget class="QLabel" name="Batch">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>355</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Nama</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNama">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Batch</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBatch">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>620</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#unggahproduk{
background-color:rgb(187,187,187,255);
border:none;
color:rgba(0,0,0,240);
border-radius:10px;
}


QPushButton#unggahproduk:hover{
background-color:rgb(255,165,0,255)
}

QPushButton#unggahproduk:pressed{
padding-left: 2px;
padding-top: 2px;
background-color:rgba(0,165,0,255);
}
</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
padding-bottom:4px;
padding-left:10px;</string>
    </property>
    <property name="placeholderText">
     <string>Search here</string>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Kategori</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputKategori">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputHarga">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>560</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Harga</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputKuantitas">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>620</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Kuantitas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBerat">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>320</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori_5">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Berat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLink">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>380</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori_6">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Link</string>
    </property>
   </widget>
   <widget class="QLabel" name="Kategori_8">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>410</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Deskripsi</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputDeskripsi">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>440</y>
      <width>380</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>1141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);
color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputidProduk">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,25);
border-radius: 10px;
padding-bottom:4px;
padding-left:10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Batch_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>295</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>idProduk</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>logoLM</zorder>
   <zorder>H_Homepage</zorder>
   <zorder>H_KatalogProduk</zorder>
   <zorder>H_Konten</zorder>
   <zorder>H_Event</zorder>
   <zorder>H_ForumDiskusi</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>Batch</zorder>
   <zorder>inputNama</zorder>
   <zorder>Kategori</zorder>
   <zorder>inputBatch</zorder>
   <zorder>editButton</zorder>
   <zorder>searchBar</zorder>
   <zorder>searchButton</zorder>
   <zorder>Kategori_2</zorder>
   <zorder>inputKategori</zorder>
   <zorder>inputHarga</zorder>
   <zorder>Kategori_3</zorder>
   <zorder>inputKuantitas</zorder>
   <zorder>Kategori_4</zorder>
   <zorder>inputBerat</zorder>
   <zorder>Kategori_5</zorder>
   <zorder>inputLink</zorder>
   <zorder>Kategori_6</zorder>
   <zorder>Kategori_8</zorder>
   <zorder>inputDeskripsi</zorder>
   <zorder>error</zorder>
   <zorder>inputidProduk</zorder>
   <zorder>Batch_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="header image.qrc"/>
 </resources>
 <connections/>
</ui>
